legal-api/report-templates/template-parts/common/effectiveDate.html

```diff
@@ -0,0 +1,13 @@
+<div class="no-page-break">
+    {% if header.reportType == 'alterationNotice' %}
+        <div class="section-title mt-4">Alteration Effective Date</div>
+        <div class="section-data mt-3">
+            The alteration is to take effect at the time thta this application is filed with the Registrar.
```

Comment:
the filing effective date/time is also in the business details, is it supposed to be here too?

Reply:
No, I am getting rid of that. Thanks.